vnext/ReactUWP/Views/TextInputViewManager.cpp

```diff
@@ -110,28 +121,65 @@ void TextInputShadowNode::createView() {
   //
   // TextChanging is used to drop the Javascript response of 'A' and expect
   // another TextChanged event with correct event count.
-  m_textBoxTextChangingRevoker = textBox.TextChanging(
-      winrt::auto_revoke, [=](auto &&, auto &&) { m_nativeEventCount++; });
+  if (m_isTextBox) {
+    m_textBoxTextChangingRevoker = control.as<winrt::TextBox>().TextChanging(
+        winrt::auto_revoke, [=](auto &&, auto &&) { m_nativeEventCount++; });
+  } else {
+    if (control.try_as<winrt::IPasswordBox4>()) {
+      m_passwordBoxPasswordChangingRevoker =
+          control.as<winrt::IPasswordBox4>().PasswordChanging(
+              winrt::auto_revoke,
+              [=](auto &&, auto &&) { m_nativeEventCount++; });
+    }
+  }
 
-  m_textBoxTextChangedRevoker =
-      textBox.TextChanged(winrt::auto_revoke, [=](auto &&, auto &&) {
-        if (auto instance = wkinstance.lock()) {
-          m_nativeEventCount++;
-          folly::dynamic eventData = folly::dynamic::object("target", tag)(
-              "text", HstringToDynamic(textBox.Text()))(
-              "eventCount", m_nativeEventCount);
-          instance->DispatchEvent(
-              tag, "topTextInputChange", std::move(eventData));
-        }
-      });
+  if (m_isTextBox) {
+    auto textBox = control.as<winrt::TextBox>();
+    m_textBoxTextChangedRevoker =
+        textBox.TextChanged(winrt::auto_revoke, [=](auto &&, auto &&) {
+          if (auto instance = wkinstance.lock()) {
+            m_nativeEventCount++;
+            folly::dynamic eventData = folly::dynamic::object("target", tag)(
+                "text", HstringToDynamic(textBox.Text()))(
+                "eventCount", m_nativeEventCount);
+            instance->DispatchEvent(
+                tag, "topTextInputChange", std::move(eventData));
+          }
```

Comment:
This part could be shared by PasswordBox and TextBox. I suggest you make it as a function